File: LocalHiringPlatform.Domain/Enums/AIIntentType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LocalHiringPlatform.Domain.Enums
{
    public enum AIIntentType
    {
        Unknown = 0,

        Greeting = 1,

        JobSearch = 2,

        CandidateQuestion = 3,

        EmployerQuestion = 4
    }
}

File: UI/localhiringplatform-ui/src/types/AI/AIIntentType.ts
export const AIIntentType = {
    Unknown: "Unknown",
    Greeting: "Greeting",
    JobSearch: "JobSearch",
    CandidateQuestion: "CandidateQuestion",
    EmployerQuestion: "EmployerQuestion",
} as const;

export type AIIntentType =
    typeof AIIntentType[keyof typeof AIIntentType];